### PR TITLE
changed classnames

### PR DESCRIPTION
**Pull request by soujanyapolavarapu**
# Description

made changes in .css  classnames to not overlap other changes.
Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/containers/ProfilePage/ProfilePage.styles.css [hunk 1]
```diff
@@ -47,70 +59,59 @@ div.quiz-results-header {
...
+  height: 5.8em;
 }
 
 .profile-career-container {
   margin-left: 4%;
 }
 
 .profile-career-container .profile-career-body {
-  margin-bottom: 40px;
+  padding: 4% 0 0 4%;
```
Comment (valmdz) [inline src/client/containers/ProfilePage/ProfilePage.styles.css:94]: Can you add some bottom padding? Something between 2% - 4%, so there is some room between the triangle and the bottom margin

File: src/client/containers/ProfilePage/ProfilePage.styles.css [hunk 2]
```diff
@@ -47,70 +59,59 @@ div.quiz-results-header {
...
+.profile-career-container div.frame {
   width: 90%;
 }
 
 .profile-career-container .profile-career-title {
   font-size: 1.5625rem;
   font-weight: bold;
-  margin: 10px 0 40px 10px;
+  margin: 0 0 6% 0;
   line-height: 0.625rem;
```
Comment (valmdz) [inline src/client/containers/ProfilePage/ProfilePage.styles.css:105]:
```suggestion
  line-height: 1.5625rem;
```

File: src/client/containers/ProfilePage/ProfilePage.styles.css (hunk 1 above)
Comment (valmdz) [reply src/client/containers/ProfilePage/ProfilePage.styles.css:94]: fx. `padding: 4% 0 2% 4%`

**Review by Enqira — APPROVED** [review]

**Review by qamarfj — APPROVED** [review]

File: src/client/containers/ProfilePage/ProfilePage.styles.css (hunk 2 above)
Comment (soujanyapolavarapu) [reply src/client/containers/ProfilePage/ProfilePage.styles.css:105]: Thankyou @valmdz i will do.

File: src/client/containers/ProfilePage/ProfilePage.styles.css (hunk 1 above)
Comment (soujanyapolavarapu) [reply src/client/containers/ProfilePage/ProfilePage.styles.css:94]: Hi @valmdz ,actually the triangle should lay beside the end of the sentence. @Poorooshka is working on it.For that sake i left it like that.Can i do it after push that triangle PR? It need any changes with some padding, thank you. 😊

**Review by TITHI1244 — APPROVED** [review]

File: src/client/containers/ProfilePage/ProfilePage.styles.css (hunk 1 above)
Comment (Poorooshka) [reply src/client/containers/ProfilePage/ProfilePage.styles.css:94]: Yeah, that's true. The PR was sent yesterday.

**Pull request merged**